tests/test_save_state.py

```diff
@@ -25,7 +25,13 @@
 @pytest.fixture
 def runner():
     """Create a coco runner."""
-    return coco_runner.Runner(CONFIG, ENDPOINTS)
+    return coco_runner.Runner(CONFIG, ENDPOINTS, reset_on_shutdown=False)
+
+
+@pytest.fixture
+def reset_runner():
+    """Create a coco runner that resets the state."""
+    return coco_runner.Runner(CONFIG, ENDPOINTS, reset_on_start=True)
```

Comment:
Confirming that you want to maintain resetting the state on shutdown for this one? =)

Reply:
By default the runner resets the state in the destructor (because it's purpose is testing only). To test the `--reset` for resetting on start of the coco server I disabled it here.

Reply:
Ah! So you want it to disable reset state on shutdown here?

Is is True on default, so I think you need to specify it.

Reply:
Oh no, you are right. This one resets on start to test the new flag. The other runner in this test doesn't reset at all.

To answer your question above: Yes, this should still reset on exit to prepare for whatever test is run next.